src/DropdownBase/test/DropdownBase.spec.js

```diff
@@ -199,6 +199,27 @@ describe('DropdownBase', () => {
     expect(await driver.isDropdownShown()).toEqual(true);
   });
 
+  it('should pass isOpen = true when opening the items list', async () => {
+    let isListOpen;
+
+    const targetDataHook = 'myOpenButton';
+    const driver = createDriver(
+      <DropdownBase {...defaultProps}>
+        {({ open, isOpen }) => {
+          isListOpen = isOpen;
+          return (
+            <IconButton dataHook={targetDataHook} onClick={open}>
+              <ChevronDown />
+            </IconButton>
+          );
+        }}
+      </DropdownBase>,
+    );
+
+    await driver.clickTargetElement(targetDataHook);
```

Comment:
Check both cases, what do you say?
```suggestion
    expect(isListOpen).toEqual(false);
    await driver.clickTargetElement(targetDataHook);
```

Reply:
But initially, it is `undefined`. Do you want to test that it is set back to `false` on a second click?

Reply:
That's not good, I added you 2 inline suggestions for this case.